cardano-testnet/src/Parsers/Cardano.hs

```diff
@@ -54,6 +54,11 @@ pCardanoTestnetCliOptions envCli = CardanoTestnetOptions
       <>  OA.help "Enable new epoch state logging to logs/ledger-epoch-state.log"
       <>  OA.showDefault
       )
+  <*> OA.flag GenerateAndRun OnlyGenerate
```

Comment:
Minor issue: I think this should be a separate command  cc: @CarlosLopezDeLara 

I wouldn't block the PR on this however.